### DISCUSSION TITLE
Vertical advance

### DISCUSSION THREAD

**Discussion by johan-overbye** [Chlumsky/msdf-atlas-gen#64] [Q&A] [answered]
I'm not seeing a vertical advance metric in my JSON file from msdf-atlas-gen. Am I missing something?

The font is Inter (Light). I had to open and export the font file from FontForge to get a kern table rather than GPOS table, in case that matters. With a font size of 1, these are the metrics I get:
```
        "emSize": 1,
        "lineHeight": 1.2102272727272727,
        "ascender": -0.96875,
        "descender": 0.24147727272727273,
        "underlineY": 0.19886363636363638,
        "underlineThickness": 0.068181818181818177
```

**Comment by Chlumsky** [comment, marked as answer]
You mean a global value for the whole font? That would be the lineHeight, right? Or do you mean for individual glyphs?

**Reply by johan-overbye** [reply]
@Chlumsky you know what, that does make sense. I was a bit confused because first of all, lineHeight here is apparently exactly descender - ascender, apart from rounding errors. Any idea why that is? Is it font-specific? Secondly I thought that sounded like too low of a vertical advance, but looking into it I understood that some characters ascend much higher than the norm. I was thinking of lowercase "f" for example, but Á is even higher for example. This is Inter on Google Fonts:

![Inter example](https://i.imgur.com/OLRDTa4.png)
